Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -349,9 +349,10 @@ open class NavigationMapView: UIView {
     }
     
     func setupGestureRecognizers() {
-        for recognizer in mapView.gestureRecognizers ?? [] where recognizer is UITapGestureRecognizer {
```

Comment:
So, mayView doesn't have tap gesture anymore?

Reply:
`MapView` keeps track of their own gestures (including taps, e.g. when selecting annotations). If I remember correctly previous implementation was added as a workaround to prevent collision with gesture recognizer, which was added by `AnnotationInteractionDelegate`.